.analysis_options

```diff
@@ -59,7 +59,6 @@ linter:
 
     # === style rules ===
     - always_declare_return_types
-    - always_put_control_body_on_new_line
```

Comment:
I think we traditionally comment out the rules instead of removing them, so we should do that here, too.

Though, I'd personally like to see this rule disappear completely, since it contradicts the Dart style guide:

> DO use curly braces for all flow control structures. There is one exception to this: if statements with no else clause that fit on one line may omit the braces.

If we split the line, then it no longer "fits" on one line, and thus must have braces.